lib/selenium-cucumber/methods/click_elements_methods.rb

```diff
@@ -1,10 +1,23 @@
 require_relative 'required_files'
 
-
 def click(access_type, access_name)
   $driver.find_element(:"#{access_type}" => "#{access_name}").click
 end
 
+def click_by_text(access_type, access_name, text)
```

Comment:
The idea behind this method is so that you can find the exact element with the text it should have and click it.  the <$driver.page_source.include? text> will return boolean if page includes that text, but it falls short of actually interacting with that element.  Also if you had some dom structure of... 

<body><h1 class='name'>Henry</h1><h1 class='name'><p>Henry</p></h1></body>

page.source.include? would return true for Henry, before it would check the second Henry, and its the second henry the user actually wants to do something with, like click perhaps. I see the especially valuable with looking at table structures. 

Hopefully by specifying the locator, selector, and the text you're looking for you won't run into a huge O(n) issue, and with the text being optional, everything behaves as it was.